compose.yml

```diff
@@ -56,6 +56,8 @@ secrets:
     file: ./.env.influxdb-admin-token
 volumes:
   influxdb-data:
+    external: true
```

Comment:
Does this mean that we have to create the volumes first externally, before running `docker compose up`?

Reply:
It would. This was not the correct fix to the issue I was seeing however, so I removed this. The config was occasionally disappearing because we need to mount to influxdb2 (I think due to a recent update?), so I also fixed the influxdb version hopefully